tests/unit/conftest.py

```diff
@@ -128,17 +128,17 @@ def storage_fixture():
     """
     Storage object fixture
     """
-    with tempfile.TemporaryDirectory() as tempdir:
-        yield LocalStorage(base_path=tempdir)
+    with tempfile.TemporaryDirectory(suffix=f"_{ObjectId()}") as tempdir:
+        yield LocalStorage(base_path=Path(tempdir))
```

Comment:
With this, do different tests still use a shared storage making the `delete_document` step in the tests necessary?

Reply:
Technically, it is not necessary.